### PR TITLE
Pull request template

### PR DESCRIPTION
**Pull request by lulf**
Adding a template based on https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/master/.github/PULL_REQUEST_TEMPLATE.md

See https://help.github.com/en/articles/creating-a-pull-request-template-for-your-repository for more info

### CODE REVIEW

File: systemtests/ansible/inventory/systemtests.inventory [hunk 1]
```diff
@@ -6,7 +6,7 @@ namespace=enmasseci
 enable_rbac=True
 api_server=True
 register_api_server=True
-service_catalog=True
+service_catalog=False
```
Comment (rgodfrey) [inline systemtests/ansible/inventory/systemtests.inventory]: Perhaps add to your template "Ensure that the pull request does not include completely unrelated changes you have accidentally checked in / pushed" ;-)

Comment (lulf) [reply systemtests/ansible/inventory/systemtests.inventory]: I was just testing reviewer quality ;) Thanks!
